### PR TITLE
add-layout-css

### PR DESCRIPTION
**Pull request by ericdarbra**

### CODE REVIEW

**Review by mikelpmc — APPROVED** [review]

**Comment by mikelpmc** [comment]
🥇

**Comment by doboapps** [comment]
👍 🔢

File: staff/ericdarbra/css/layout/styles/main.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+header {
+    background-color: blue;
+    animation-name: test;
+    animation-duration: 3s;
+    width: 100%;
+    height:20%;
+    text-align: center;
+
+}
+@keyframes test{
```
Comment (manuelbarzi) [inline staff/ericdarbra/css/layout/styles/main.css]: TIP ```test```has NO semantic. may you rename it to a more semantic one? same for test2, test3....

File: staff/ericdarbra/css/mediaquery/styles/desktop.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
+* {
```
Comment (manuelbarzi) [inline staff/ericdarbra/css/mediaquery/styles/desktop.css]: TIP some css rules repeat in mobile and desktop. may you group them in main.css?

File: staff/ericdarbra/cv/styles/main.css [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    #formacion p{
+        max-width: 1440px;
+        margin:0 auto;
+        line-height: 40px;
+    }
+
+    #competencias {
+        margin: 0 auto;
+        font-family: ubuntu;
+        /*background: #f5f5f5;*/
```
Comment (agandia9) [inline staff/ericdarbra/cv/styles/main.css:42]: caution with comments... delete it before push!

File: staff/ericdarbra/cv/styles/main.css [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+            margin-bottom: 25px;
+        }
+        footer p {
+            margin: 30px;
+            order: 2;
+            font-size: 105%;
+            line-height: 25px;
+            max-width: 1090px;
+        }
+        footer section{background: #009ee4;margin: 0 auto;
```
Comment (agandia9) [inline staff/ericdarbra/cv/styles/main.css:78]: correct identation here...

**Comment by ericdarbra** [comment]
giphy

**Pull request merged**